tests/devapps/DesktopTestApp/Program.cs

```diff
@@ -18,7 +18,7 @@ static void Main()
             Application.EnableVisualStyles();
             Application.SetCompatibleTextRenderingDefault(false);
 
-            ServicePointManager.SecurityProtocol = SecurityProtocolType.Tls12 | SecurityProtocolType.Ssl3;
```

Comment:
@bgavrilMS new analyzer complains about using this as it is outdated. Is this still needed?